src/h5web/dataset-visualizer/DatasetVisualizer.tsx

```diff
@@ -28,7 +29,10 @@ interface Props {
 
 function DatasetVisualizer(props: Props): JSX.Element {
   const { dataset } = props;
-  const dimensionMapping = { x: 0, y: 1 };
+  const [dimensionMapping, setMapping] = useState<DimensionMapping>({
+    x: 0,
+    slicingIndices: { 1: 0 },
+  });
```

Comment:
This should be replaced by something more clever that depends on the dataset and the activeVis.